CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Versioned Releases
 
-## 2.0.0 (Jan 13, 2020)
+## 2.1.0 (Feb 19, 2021)
```

Comment:
Also need to update `UnityLibraryVersion` in `Amplitude.cs`. Also we shouldn't include this change within this PR.

The right way to do it is to create a separate PR called `Prepare release x.x.x` by only updating the version and CHANGELOG. Please read the commit history.